### DISCUSSION TITLE
New error after installation

### DISCUSSION THREAD

**Discussion by 1000camels** [thegetty/quire#99] [Get Help] [answered]
Hi, I am working with a new client who is experiencing an error after installing quire. Has anyone seen this error before?

 > quire preview
Launching preview server
ℹ Navigate to http://localhost:1313 to see your changes.
ℹ Press Control+C or type "quire stop" to stop the preview.
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error: spawn /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/hugo-bin/vendor/hugo ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:465:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/hugo-bin/vendor/hugo',
  path: '/usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/hugo-bin/vendor/hugo',
  spawnargs: [
    'server',
    '--config=config.yml,config/environments/dev.yml',
    '--watch'
  ]
}

**Comment by geealbers** [comment]
It looks like it's not finding Hugo? @hbalenda any thoughts? My dummy suggestion would be for them to try running `npm install --global @thegetty/quire-cli` again and seeing if that shakes something loose. And if it doesn't, providing us the whole output from that new install attempt, where we should be able to see a little more.

@1000camels what system is your client on, macOS?

**Comment by 1000camels** [comment]
Yes, macOS

**Comment by hbalenda** [comment]
@geealbers I haven't been able to reproduce the error, but I just published the latest patch version of quire to npm, which included a fix related to installing dependencies. @1000camels please have the client re-install with `npm install -g @thegetty/quire-cli`, confirm the version is `0.19.1` by running `quire -V` and then try creating a new project again. Hopefully this should resolve the issue. But if not, please have them provide us with the install output, and then we can take a closer look. Thank you!

**Comment by emptyhut** [comment, marked as answer]
Darcy was helping me install Quire. My problem was an earlier installation of Nodejs, npm, and yarn built last year via MacPorts then soon forgotten. I was using the forgotten versions from MacPorts due to $path.  
After uninstalling all three the installation instructions worked as given.
